content/authors/Ngopnang/_index.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think the content of this file is missing. It should look like the template, here is another copy of the template. Please try putting that in your `_index.md` file and updating the values (your name etc.)
```
---
# Display name
title: Your Name Goes Here

# Username (this should match your github username)
authors:
- YourGitHubUsername

# Role/position e.g. Graduate student
role: Postdoctoral Fellow

# Organization membership
Organizations/Affiliations:
organizations:
- name: Your Current University
  url: https://www.google.ca

# A list of your interests, add as many as you want
interests:
- Scientific Machine Learning
- General Purpose Data Science
- Mathematical Biology

# Your educational history, add as many as you want
education:
  courses:
  - course: PhD - Applied Mathematics
    institution: Univerisity of Alberta
  - course: BSc - Mathematics
    institution: Universidad of British Columbia

# Social/Academic Networking, delete any items you don't want to use
social:
- icon: github
  icon_pack: fab
  link: https://github.com/pimsmath
- icon: twitter
  icon_pack: fab
  link: https://twitter.com/carlosshigoto
#- icon: linkedin
#  icon_pack: fab
#  link: https://www.linkedin.com/in/carloscontrerasc/

user_groups:
- committee: members
  role: Project 1 member

tags:
- '2021'
---
This is a freeform text area which supports markdown syntax. Fill in a couple of
sentences describing yourself.
```

Reply:
Dear Ian,

I don't know what I am doing wrong but I used the template and updated all values as you asked.
See the attached image for an insight of my _index.md file.
<img width="1440" alt="_index md file" src="https://user-images.githubusercontent.com/87891164/129396741-f8a761de-e411-478a-9d1d-1f760d7d9711.png">